src/plugins/intel_cpu/src/graph_dumper.cpp

```diff
@@ -245,6 +245,11 @@ void serializeToXML(const Graph& graph, const std::string& path) {
     }
 
     std::string binPath;
+#    ifdef _WIN32
+    binPath = "NUL";
```

Comment:
Let's create a platform.h inside cpu utils folder and put it there.
Let's call it a NULL_STREAM or something.
Could you please update intel_cpu/src/utils/ngraph_transformation.hpp:79, the same idea is used there.

Reply:
Done